src/components/CreatureDetail.js

```diff
@@ -60,6 +62,7 @@ class CreatureDetail extends Component {
         {/* <p>Movement: {this.state.creatureMove}</p> */}
         <p>XP: {creatureInfo.xp}</p>
         <p>Actions: <br></br>{this.state.creatureActions}</p>
+        <button className="add-fav-btn" onClick={() => addFavorite(creatureInfo.name, creatureInfo.challenge_rating, `creatures/${creatureInfo.index}/details`)}>Add Favorite</button>
```

Comment:
Calling this function from the other component seemed like the most straight-forward way to access it, but let me know if you see any ways to improve this.